ra-optimisation/search-tree-rao/src/main/java/com/powsybl/openrao/searchtreerao/linearoptimisation/algorithms/BestTapFinder.java

```diff
@@ -45,19 +45,14 @@ private BestTapFinder() {
      * If virtual costs are an important part of the optimization, it is highly recommended to use APPROXIMATED_INTEGERS
      * taps in the linear optimization, rather than relying on the best tap finder to round the taps.
      *
-     * @return a map containing the best tap position for every PstRangeAction that was optimized in the linear problem
      */
-    public static RangeActionActivationResult round(RangeActionActivationResult linearProblemResult,
+    public static void round(RangeActionActivationResult linearProblemResult,
```

Comment:
it's generally bad design to ask a method to change one of its arguments. try to keep roundedResult as a returned value